scripts/manual_validation_data_processing.R

```diff
@@ -0,0 +1,275 @@
+}
+
+if (setequal(data_transformed$completion_month_year_reg2, drks_ctgov_data$completion_month_year_reg1)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+#comparison completion date type
+if (setequal(data_transformed$completion_date_type_reg1, drks_ctgov_data$completion_date_type_reg2)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+if (setequal(data_transformed$completion_date_type_reg2, drks_ctgov_data$completion_date_type_reg1)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+#comparison recruitment status
+if (setequal(data_transformed$recruitment_status_reg1, drks_ctgov_data$recruitment_status_reg2)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+if (setequal(data_transformed$recruitment_status_reg2, drks_ctgov_data$recruitment_status_reg1)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+#comparison overall recruitment status
+if (setequal(data_transformed$overall_recruitment_status_reg1, drks_ctgov_data$overall_recruitment_status_reg2)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+if (setequal(data_transformed$overall_recruitment_status_reg2, drks_ctgov_data$overall_recruitment_status_reg1)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+#comparison has summary results main
+if (setequal(data_transformed$has_summary_results_reg1_main, drks_ctgov_data$has_summary_results_reg2_main)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+if (setequal(data_transformed$has_summary_results_reg2_main, drks_ctgov_data$has_summary_results_reg1_main)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+#comparison has summary results sensitivity 
+if (setequal(data_transformed$has_summary_results_reg1_sensitivity, drks_ctgov_data$has_summary_results_reg2_sensitivity)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+if (setequal(data_transformed$has_summary_results_reg2_sensitivity, drks_ctgov_data$has_summary_results_reg1_sensitivity)) {
+  print("Both columns contain the same values")
+} else {
+  print("The columns do not contain the same values")
+}
+
+#### RE-JOINING FINAL DATASET FOR ANALYSIS ####
+joined_data <- bind_rows(euctr_data, data_transformed)
+
+### EXPORT FINAL DATASET
+write.csv(joined_data, "~/Desktop/Research group Strech/intovalue-crossreg/data/manual_validation_processed.csv",
```

Comment:
@elisabascunan please also update this with the `{here}` package as you do when you read the file, thanks!